src/constants.js

```diff
@@ -59,6 +59,9 @@ export const CONFIG_DIRECTORY = getDirectory('config');
 export const LINK_REGISTRY_DIRECTORY = path.join(CONFIG_DIRECTORY, 'link');
 export const GLOBAL_MODULE_DIRECTORY = path.join(CONFIG_DIRECTORY, 'global');
 
+export const POSIX_GLOBAL_PREFIX = '/usr/local';
```

Comment:
>Maybe the website docs can be updated with how we get the prefix so that regular users can find the info easily, and why we don't mimic NPM.

I think this is a great idea! @Daniel15 any suggestions? I can file an issue for this so we don't forget.